app/Livewire/Project/New/PublicGitRepository.php

```
@@ -363,10 +363,24 @@ public function submit()
+                        $gitUrl,
+                        $this->git_branch,
+                        $this->base_directory,
+                        $destination->server->id,
+                        auth()->user()->currentTeam()->id
+                    );
+                    if ($config) {
+                        $application->setConfig($config, fromRepository: true);
+                        session()->flash('success', 'coolify.json configuration detected and applied.');
+                    }
+                } catch (\Exception $e) {
+                    \Log::warning('coolify.json: Failed to apply config - '.$e->getMessage());
```

PHPMD (2.15.0)

Avoid using static access to class '\Log' in method 'submit'. (undefined)

(StaticAccess)